content/en/docs/getting-started/examples.md

```diff
@@ -15,6 +15,26 @@ Use the [`openshift-e2e-aws`](https://steps.ci.openshift.org/workflow/openshift-
     workflow: openshift-e2e-aws
 {{< / highlight >}}
 
+# How do I write a simple "Execute this command in a container" test?
+
+Use a container test. Container tests are set up to always contain the source code, either by
+explicitly cloning it if they use an image that is in `base_images` or implicitly if they reference
+a `pipeline` image like `src`.
+
+{{< highlight yaml >}}
+base_images:
+  golangci-lint:
+    namespace: ci
+		name: golangci-lint
+		tag: v1.37.1
+tests:
+- as: lint
+	commands: golangci-lint run ./...
+	container:
+		from: golangci-lint
+		clone: true # Defaults to "true", set to "false" if you do not want your source code to be present.
```

Comment:
Technically the statement is not 100% accurate, as it will default to `false` when a pipeline image is used, but IMHO the important part to understand is `By default your code will be there` and not how exactly this is achieved.

Reply:
(And I am not sure if there is a valid usecase for not wanting the code to be present in a container test)

Reply:
I agree with your thinking and yes, I don't think anyone wants to run presubmits without the code in there